analyzer/src/funTest/assets/projects/synthetic/gomod-expected-output.yml

```diff
@@ -22,6 +22,8 @@ project:
       linkage: "PROJECT_STATIC"
     - id: "GoMod::github.com/fatih/color:v1.13.0"
       linkage: "PROJECT_STATIC"
+    - id: "GoMod::github.com/go-playground/assert/v2:v2.0.1"
```

Comment:
Question regarding the commit message, when you say "which illustrates the former" it means that for this library the source code is in a "v2" folder, and respectively that for the library added in the next commit it is not?
I got a bit confused, because in both cases ORT does not set the VCS path to "v2", so maybe you could make it a bit clearer, that what you illustrate is that ORT ignores that suffix (probably because you plan to fix this later on?).